src/widget/taskBar.js

```diff
@@ -79,7 +81,7 @@ var TaskBar = GObject.registerClass(
             //if you change the class before animate the indicator there is an issue for retrieving the item.x
 
             this._animateActiveIndicator();
-            nextItem.add_style_class_name('active');
+            nextItem.setActive(true);
```

Comment:
I am getting this error, and I think it is related to the change above:

```
set 28 12:41:26 miix320-pop gnome-shell[158416]: JS ERROR: Exception in callback for signal: tileable-focus-changed: TypeError: nextItem.setActive is not a function
                                                 onFocusChanged@/home/lfom/.local/share/gnome-shell/extensions/material-shell@papyelgringo/src/widget/taskBar.js:84:22
```

Also, I have noticed the the AppLauncher icon ("+") is not centered anymore.